pkg/service/nodes.go

```diff
@@ -77,7 +77,9 @@ func drainNode(kubeClient kubernetes.Interface, node *v1.Node, timeout, retryInt
 	}
 
 	for retryAttempts > 0 {
-		err = drainNodeUtil(node, int(timeout), kubeClient)
+		// create a copy of the node obj, since RunCordonOrUncordon() modifies the node obj
+		nodeCopy := node.DeepCopy()
+		err = drainNodeUtil(nodeCopy, int(timeout), kubeClient)
 		if err == nil {
 			log.Info("drain suceeded")
```

Comment:
Can you also fix the spelling error and change to `succeeded`?

Reply:
Please help to add the node name and instance id in both `retry drain` and `drain succeeded` logs.
it will help debugging.

Reply:
fixed both. please review again.